l3kernel/l3tl.dtx

```diff
@@ -672,6 +671,29 @@
 %   \end{texnote}
 % \end{function}
 %
+% \begin{function}[added = 2025-02-02, EXP]
+%   {
+%     \tl_trim_left_spaces:n, \tl_trim_left_spaces:V, \tl_trim_left_spaces:v,
+%     \tl_trim_left_spaces:e, \tl_trim_left_spaces:o,
+%     \tl_trim_right_spaces:n, \tl_trim_right_spaces:V, \tl_trim_right_spaces:v,
+%     \tl_trim_right_spaces:e, \tl_trim_right_spaces:o
+%   }
+%   \begin{syntax}
+%     \cs{tl_trim_left_spaces:n} \Arg{token list} \\
+%     \cs{tl_trim_right_spaces:n} \Arg{token list}
```

Comment:
Maybe only one syntax case is needed?

Reply:
yes, we usually don't show all variations in the syntax examples and this here is obvious